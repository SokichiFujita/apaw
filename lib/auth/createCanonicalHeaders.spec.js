const createCanonicalHeaders = require('./createCanonicalHeaders');

describe('createCanonicalHeaders', () => {
  it('should create a formatted string of header key/values', () => {
    const headers = {
      'Content-Encoding': 'amz-1.0',
      'X-Amz-Date': '20200308T123600Z',
    };
    const canonicalHeaders = createCanonicalHeaders(headers);
    expect(canonicalHeaders).toMatchSnapshot();
  });
});
