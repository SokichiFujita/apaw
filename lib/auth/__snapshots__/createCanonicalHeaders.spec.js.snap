// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createCanonicalHeaders should create a formatted string of header key/values 1`] = `
"content-encoding:amz-1.0
x-amz-date:20200308T123600Z
"
`;
